tempMonApp/src/tempMon.c

```diff
@@ -405,3 +410,57 @@ cpu_thermal_init(int hwmon_inst, struct temp_state* state)
   state->fd = open(file, O_RDONLY);
   return state->fd < 0 ? -1 : 0;
 }
+
+/******************************************************************************
+ * IOC shell function registration
+ *****************************************************************************/
+
+static void
+hwmonStatusCallFunc(const iocshArgBuf* args)
```

Comment:
If this function re-indexes stuff, it could have a more descriptive name instead of hwmonStatus.